gap/PolygonalComplexes/modification.gd

```diff
@@ -1206,6 +1235,55 @@ DeclareOperation( "TetrahedralExtension", [IsSimplicialSurface, IsPosInt] );
 DeclareOperation( "EdgeTurn", [IsSimplicialSurface, IsPosInt,IsPosInt] );
 #! @EndGroup
 
+#! @BeginGroup BuildingBlocks
+#! @Description
+#! Return the building blocks of a vertex-faithful simplicial sphere.
+
+#! Given two closed simplicial surfaces with euler-characteristic 2 a new surface
+#! can be constructed by computing the connected-face sum 
+#! (<Ref Subsect="ConnectedFaceSum"/>) of this surfaces.
+#! This gives rise to a waist of length 3 in the resulting surface.
+#! Applying this procedure for example to two tetradron with any two faces 
+#! gives rise to the double tetrahedron.
+ 
+#! Given a vertex-faithful simplicial sphere there exists a finite number of 
+#! simplicial spheres without waists of length 3, so that iteratively computing the 
+#! connected-face-sum of those surfaces gives rise to <K>surface</K>, up to 
+#! isomorphism. This surfaces which are unique up to isomorphism are called the 
+#! building blocks of the simplicial sphere <K>surface</K>. So the double tetrahedron
+#! has two building blocks, namely two tetrahedron.
+#! Since building blocks only exist for vertex-faithful spheres the function returns
+#! <K>fail</K>, if <K>surface</K> does not satisfy this property. 
+#! @BeginExampleSession
```

Comment:
It would be great to have some explanations to the examples and to rename s and ss in better names for example doubleTetra for the surface s.